provisioning/device/src/Utilities/CertificateInstaller.cs

```diff
@@ -20,7 +21,10 @@ internal static class CertificateInstaller
         /// for more information.
         /// </remarks>
         /// <param name="certificates">The certificate chain to ensure is installed.</param>
-        public static void EnsureChainIsInstalled(X509Certificate2Collection certificates)
+        /// <param name="certStore">For unit testing to allow for mocking.</param>
+        internal static void EnsureChainIsInstalled(
+            X509Certificate2Collection certificates,
+            ICertificiateStore certStore = default)
```

Comment:
The client won't pass this parameter, but a unit test can, with a mock of `ICertificateStore`. By doing so it can pass different certificates and control whether the code will try to add it to the store or not.